pkgs/applications/editors/pulsar/001-patch-wrapper.patch

```diff
@@ -0,0 +1,28 @@
+--- ./resources/pulsar.sh	2023-03-16 04:11:14.000000000 +0100
++++ ./resources/pulsar.sh	2023-03-24 14:37:13.468813964 +0100
+@@ -123,22 +123,10 @@
+ elif [ $OS == 'Linux' ]; then
+   SCRIPT=$(readlink -f "$0")
+ 
+-  PULSAR_PATH="/opt/Pulsar/pulsar"
++  PULSAR_DEFAULT="/opt/Pulsar/pulsar"
++  PULSAR_PATH="${PULSAR_PATH:-$PULSAR_DEFAULT}"
+ 
+-  #Will allow user to get context menu on cinnamon desktop enviroment
+-  #Add a check to make sure that DESKTOP_SESSION is set before attempting to grep it
+-  #expr substr is expecting 3 arguments string, index, length
+-  #If grep doesnt find anything is provides an empty string which causes the expr: syntax error: missing argument after '8' error - see pulsar-edit/pulsar#174
+-  #Im also not quite sure why they used grep instead of simply [ "${DESKTOP_SESSION}" == "cinnamon" ]
+-  if [ -n "${DESKTOP_SESSION}" ] && [ "$(expr substr $(printenv | grep 'DESKTOP_SESSION=') 17 8)" == "cinnamon" ]; then
+-    #This local path is almost assuredly wrong as it shouldnt exist in a standard install
```

Comment:
@bobby285271 are we even ħandling nemo actions provided by packages currently? this would be one package providing them. ideally nemo should just pick them up from some location, /run/current-system/sw/share/nemo/actions (then the file would need to be in share/nemo/actions in this package)

Reply:
Nemo picks up actions from all `XDG_DATA_DIRS`, `/run/current-system/sw/share/nemo/actions` *should* work I think.

https://github.com/linuxmint/nemo/blob/5.6.4/libnemo-private/nemo-action-manager.c#L146-L159

Reply:
@COLAMAroro you can then cp (or better mv) the file into share/nemo/actions in the derivation (it needs to be removed from the script anyways). it'll appear in /run/current-system/sw/share automatically

Reply:
Turns out, the wrapper script does the install... But it's not provided in the pre-built tarball. I just went out of my way to get the upstream action and install manually

Reply:
tested, works great